mozdownload/scraper.py

```diff
@@ -124,11 +124,10 @@ def __init__(self, destination=None, platform=None,
 
         self.destination = destination or os.getcwd()
 
-        if not locale:
-            if application in APPLICATIONS_MULTI_LOCALE:
-                self.locale = 'multi'
-            else:
-                self.locale = 'en-US'
+        if application in APPLICATIONS_MULTI_LOCALE:
+            self.locale = 'multi'
+        elif not locale:
```

Comment:
To better understand the logic I would suggest that you check here for `elif locale` and fallback to `en-US` in the else case.